assets.tf

```diff
@@ -77,7 +78,7 @@ data "template_file" "kubeconfig-kubelet" {
     ca_cert      = "${base64encode(var.ca_certificate == "" ? join(" ", tls_self_signed_cert.kube-ca.*.cert_pem) : var.ca_certificate)}"
     kubelet_cert = "${base64encode(tls_locally_signed_cert.kubelet.cert_pem)}"
     kubelet_key  = "${base64encode(tls_private_key.kubelet.private_key_pem)}"
-    server       = "${format("https://%s:%s", element(var.api_servers, 0), var.apiserver_port)}"
+    server       = "${format("https://%s:%s", element(var.api_servers_private, 0), var.apiserver_port)}"
```

Comment:
This currently affects all deployment targets. We need to find a way to control this from `variables.tf`.